release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -425,6 +425,11 @@ With this release, you can configure the IPv4 join subnet that is used internall
 
 For more information, see xref:../installing/installing_aws/installation-config-parameters-aws.adoc#installation-configuration-parameters-network_installation-config-parameters-aws[Network configuration parameters].
 
+[id="ocp-4-18-installation-and-update-gcp-c4_{context}"]
+==== Installing a cluster on {gcp-short} using the C3 and C4/C4A machine series
+
+With this release, you can deploy a cluster on {gcp-short} using the C3, C4 and C4A machine series for compute or control plane machines. The supported disk type of these machines is `hyperdisk-balanced`. If you use an instance type that requires Hyperdisk storage, all of the nodes in your cluster must support Hyperdisk storage, and you must change the default storage class to use Hyperdisk storage. For more information about configuring machine types, see xref:../installing/installing_gcp/installation-config-parameters-gcp.adoc#installation-configuration-parameters-gcp[Installation configuration parameters for GCP].
```

Comment:
The supported disk type of C4 and C4A machine series is `hyperdisk-balanced`, but C3 machine series does support disk types `pd-balanced` and `pd-ssd`, along with `hyperdisk-balanced` (see Google [doc](https://cloud.google.com/compute/docs/general-purpose-machines#c3_disks)).

Reply:
FYI from the above Google doc: 
`C3 bare metal instances support only Hyperdisk storage. You can't use Local SSD disks with bare metal instances.`